jsk_pcl_ros/scripts/download_pcd_data.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PCD_DATA_DIR=$(rospack find jsk_pcl_ros)/pcds
+
+echo "Downloading pcd data"
+
+# download pcd data for drc drill task
+wget "http://www.jsk.t.u-tokyo.ac.jp/~ueda/dataset/2015/02/drill.pcd" -O $PCD_DATA_DIR/drill.pcd
```

Comment:
But maybe you can rebase from this PR: https://github.com/jsk-ros-pkg/jsk_recognition/pull/1627